cmd/app/options/options.go

```diff
@@ -48,13 +48,13 @@ func (o *Options) AddFlags(cmd *cobra.Command) {
 	usageFmt := "Usage:\n  %s\n"
 	cols, _, _ := term.GetSize(0)
 	cmd.SetUsageFunc(func(cmd *cobra.Command) error {
-		fmt.Fprintf(cmd.OutOrStderr(), usageFmt, cmd.UseLine())
+		_, _ = fmt.Fprintf(cmd.OutOrStderr(), usageFmt, cmd.UseLine())
```

Comment:
Don't ignore the error being returned. You can ignore `n`. 
- https://pkg.go.dev/fmt#Fprintf 

So something like this:

```suggestion
		if _, err := fmt.Fprintf(cmd.OutOrStderr(), usageFmt, cmd.UseLine()); err != nil {
				return err
		}
```